content/ros/ROS_Nodes_Topics_Messages/home_work/README.md

```diff
@@ -0,0 +1,56 @@
+# Homework assignment
+
+In this assignment, you are asked to create a node, which for a given 
```

Comment:
Minor rephrasing/punctuation changes:

`In this assignment, you are asked to create a node which, for a given coordinates and a given map, returns whether the given location is occupied (i.e. whether there is an obstacle), free, or unknown.`

Reply:
done